src/network/network.h

```diff
@@ -20,18 +20,17 @@
     OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
     THE SOFTWARE.
 */
+
 #ifndef SLIMEVR_NETWORK_H_
 #define SLIMEVR_NETWORK_H_
 
+#include "WiFiHandler.h"
 #include "globals.h"
-#include "wifihandler.h"
-#include "udpclient.h"
 #include "packets.h"
-#include "wifiprovisioning.h"
+#include "udpclient.h"
 
 namespace Network {
-    void update(Sensor * const sensor, Sensor * const sensor2);
-    void setUp();
+    void update(Sensor* const sensor, Sensor* const sensor2);
```

Comment:
is the pointer const or is the sensor const? I believe this makes the pointer const but not the underlying `Sensor`. Is that intended?

Reply:
@Eirenliel

Reply:
Feel free to mark as resolved if Eiren doesn't get back to you before the PR is ready for merge